src/AbstractDatabase.php

```diff
@@ -101,12 +101,12 @@ private function loadDatabase($databaseFile)
         $isoNumber = $this->getISONumber();
 
         // add gettext domain
-        bindtextdomain(
+        \bindtextdomain(
             $isoNumber,
```

Comment:
Why?

Reply:
It seems that it's the internal PHP function and we have add the ```\``` to avoid the namspace issue.

Reply:
What issues soy you try to fix?

Reply:
You think someone creates function in this namespace with same name??

Reply:
If we remove the ```\```, the current function name will be the ```Sokil\IsoCodes\bindtextdomain```.

Reply:
Really? So how it currently works? ))))

Reply:
It's the same as [this](https://github.com/peter279k/php-isocodes/blob/6421cfc36c524609382da94550d3383465e8e763/src/AbstractDatabase.php#L205).